core/base/lin_op.hpp

```diff
@@ -340,7 +340,7 @@ class LinOp : public EnableAbstractPolymorphicObject<LinOp> {
  * auto cg_factory = solver::Cg::Factory<>::create()
```

Comment:
Replace with `build()`? Also There probably aren't `<>` for `Factory` but for `Cg`